ci/experimental/docker/scripts/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Please don't add `docker/scripts` directory, this script is in each of the docker directories already